src/test/scala/chiseltest/backends/verilator/VerilatorClockPokeTest.scala

```diff
@@ -51,7 +50,7 @@ class VerilatorClockPokeTest extends AnyFlatSpec with ChiselScalatestTester {
     }
   }
 
-  it should "clock-as-bool-verilator-clock-poke" in {
+  it should "clock-as-bool-verilator-clock-poke" ignore { // TODO: re-add multi clock support
```

Comment:
Is clock poking as a bool just not supported at all? Is this a limitation of our API or the underlying simulator?

Reply:
Clock poking as a bool only makes sense if there is a way to advance the simulation time. In this example that is done by advancing the "main" clock. However, I am not a fan of having such an "asymmetrical" API, where one clock needs to be stepped in order to advance time for the other clock. Since no-one seems to use this so far, I would like to punt the decision on how to support multi-clock to the future.

Wrt. simulator support: poking top-level clock inputs should be fine for verilator and treadle and probably also for VCS. Essent won't support it, but then it also just doesn't support multi-clock at all (Scott said that adding multi-clock support is an eventual goal).